e2e-tests/playwright/tests/functional/channels/gif_picker/gif_picker_search.spec.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect, test} from '@e2e-support/test_fixture';
+
+test('MM-XXX Should search, select and post correct Gif when Gif picker is opened from center textbox', async ({
```

Comment:
Do you have the PR to test management repo or test ID available to replace `MM-XXX`?

Reply:
Added https://github.com/mattermost/mattermost-test-management/pull/111

Reply:
Done ✅